euphoria-spark/src/main/java/cz/seznam/euphoria/spark/SparkStorageProvider.java

```diff
@@ -20,7 +20,8 @@
 import cz.seznam.euphoria.core.client.operator.state.StorageProvider;
```

Comment:
unused MemListStorage subclass

Reply:
That was probably replaced with the `FsSpillingListStorage`. Dropped that.